launcher/agent.go

```diff
@@ -28,7 +27,7 @@ type principalIDTokenFetcher func(audience string) ([][]byte, error)
 type AttestationAgent struct {
 	tpm              io.ReadWriteCloser
 	akFetcher        tpmKeyFetcher
-	client           servpb.AttestationVerifierClient
+	client           verifier.Client
 	principalFetcher principalIDTokenFetcher
 	cosCel           cel.CEL
 	logger           *log.Logger
```

Comment:
Are we still using this logger parameter?

Reply:
yes for one line. I've removed that line because logging can be done in the client and so removed the logger as well.